docs/BBA-EN.md

```diff
@@ -0,0 +1,189 @@
+# Binary Byzantine Agreement
+
+## Overview
+
+Binary Byzantine Agreement tries to solve consensus problem in asynchronous distributed systems where processes or node can commit Byzantine failures. A process can have a Byzantine behavior when it arbitrary deviates from its intended behavior, it then commits a Byzantine failure.
+
+This bad behavior can be intentional or simply the result of temporary fault that altered the its normal behavior in unpredictable way. One example of bad behavior is crash of process.
+
+
+
+## Binary Byzantine consensus problem
+
+The Binary Byzantine consensus problem is that each correct process `p(i)` proposes a value `v(i)` (0 or 1). And each of members in network has to decide a value such that the following properties are satisfied:
+
+* A decided value was proposed by a correct process
+* No two correct processes decide different values
+* A correct process decides at most once.
+* Each correct process decides, and terminate consensus
+
+These properties states that value proposed only by faulty node can be decided. In other words, if all correct processes propose the same value `v`, the value `v'` cannot be decided. 
```

Comment:
I can't understand this statement well.
```In other words, if all correct processes propose the same value `v`, the value `v'` cannot be decided. ```
Why is it?
as I think, if all correct processes propose the same value `v`, the value `v` can be decided.

Reply:
v' ! not v :)